src/controllers/document_upload/document.upload.controller.ts

```diff
@@ -159,7 +159,7 @@ const getUploadFinishedCallback = (req: Request,
 
       if (e.status === HttpStatusCodes.UNSUPPORTED_MEDIA_TYPE) {
         return await
-          displayError(res, UploadErrorMessages.INVALID_MIME_TYPES, uploadResponderStrategy, attachments);
+        displayError(res, UploadErrorMessages.INVALID_MIME_TYPES, uploadResponderStrategy, attachments);
```

Comment:
The indentation looks wrong here - it's a line continuation, but now that the indentation has gone, it starts to look like two separate lines.

Reply:
Not sure how to refine this, there does not seem to be any parameter to acknowledge line continuation.

Reply:
Bit stuck here not sure how to refine the indentation rule to get it to acknowledge continued expressions.

Reply:
if we just return the indentation to as it was on this line, will eslint be ok with it?  or just fit it all on 1 line, looks like it might fit.

Reply:
Just googled the eslint indentation rules and also can't find a switch to control this. Perhaps try as @markpit suggests above, if not, I guess we live with this.

Reply:
Tried this, but the rule complains so we'll have to keep it as it is for now.